lib/checkWebHook.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var bufferEq = require('buffer-equal-constant-time');
+var Cesium = require('cesium');
+var crypto = require('crypto');
+
+var Check = Cesium.Check;
+var defined = Cesium.defined;
+
+var Settings = require('./Settings');
+module.exports = checkWebHook;
+
+function checkWebHook(req, res, next) { // eslint-disable-line no-unused-vars
+    Check.typeOf.object('req', req);
+
+    if (req.method !== 'POST') {
+        throw new Error('Request was not POST');
+    }
+
+    // check header fields
+    var id = req.headers['x-github-delivery'];
+    if (!defined(id)) {
+        throw new Error('No id found in the request');
```

Comment:
All of these errors should be in the form of 

```js
next(new Error('No id found in the request'));
return;
```

In general, don't throw from route handlers.